src/index.html

```diff
@@ -5,8 +5,460 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <img class="header__img" src ="images/logo.png"
+          alt="logo"href="#home">
+        <nav class="nav">
+          <ul class="nav__list"></ul>
+          <li class="nav__li-item">
+            <a class="nav__link" href="#video"
+            >
+              Video
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a class="nav__link" href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a class="nav__link" href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a class="nav__link" href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a
+              class="nav__link"
+              href="#laptops&computers"
+              data-qa="nav-hover"
+            >
+              Laptops &#38 computers
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a class="nav__link" href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a class="nav__link" href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__li-item">
+            <a class="is-active" href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="main">
+      <div class="main__catalog">
+        <div class="container">
```

Comment:
you do not need this container

Reply:
I tried to delete it and move styles, but after everything is broken